package.json

```diff
@@ -26,6 +26,7 @@
     "ember-cli-app-version": "^1.0.0",
     "ember-cli-dependency-checker": "^1.3.0",
     "ember-cli-eslint": "^3.0.0",
+    "ember-cli-fastboot": "^1.0.2",
```

Comment:
Primarily to ensure this approach worked in fastboot.  Means that we can run `ember s` and disable JS then inspect the `head` element.  I can remove if that is a problem